test/OpenSpaceToolkit/Astrodynamics/Trajectory/State/CoordinatesBroker.test.cpp

```diff
@@ -346,4 +347,55 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_State_CoordinatesBroker, Extrac
 
         EXPECT_ANY_THROW(broker.extractCoordinates(fullCoordinatesVector, subset_4));
     }
+
+    {
+        CoordinatesBroker broker = CoordinatesBroker();
+        broker.addSubset(subset_1);
```

Comment:
Didn't know you could make these fixture like objects like subset_1 in cpp. Very cool. Should its name be more descriptive than _1 for test legibility perhaps?

Reply:
Yeah, they are effectively just C++ class variables. All of these test functions are considered part of the test class because they use `TEST_F` in GTest rather that `TEST`. It's pretty much just a class method accessing a class variable. `this->subset_1` would be equivalent, but you don't necessarily have to specify `this` in C++ as long as there's no naming collision.

Good looking out. In this case, the subsets don't represent anything physical (the names are like "S1", "S2", etc.), so I think the naming is okay.